src/signals/incident-management/containers/FilterTagList/index.js

```diff
@@ -31,11 +28,11 @@ const renderTag = (key, tagKey, mainCategories, list) => {
 
   let display = (found && found.value) || key;
 
-  if (!display || ignoredTags.includes(tagKey)) {
+  if (!display) {
     return;
   }
 
-  if (isDate(display)) {
+  if (tagKey === 'incident_date') {
```

Comment:
There will be other fields values in the future that will contain dates, take for instance the proposed change to the date field where a range is possible. You should still be checking on the contents of a field otherwise we'll end up with a whitelist that we'll actively have to maintain, which might lead to confusing and unexpected visual presentations of the tags list.

Reply:
I agree, but isDate uses Date.parse('weesperstraat 113'). that will returns a correct date. but it is not a date. how to check for this then?

Reply:
I think the `isDate` function is too limited. We can replace it with [lodash.isdate](https://www.npmjs.com/package/lodash.isdate)

Reply:
de lodash isDate is niet geschikt omdat hij  alle string values afkeurt als datum.

volgens mij moeten we toch maar op key de datum formatten, dus dat ga ik er maar terug in bouwen